code/src/App.js

```diff
@@ -1,9 +1,70 @@
-import React from 'react'
+import React, { useState, useEffect }from 'react'
+import PostThoughts from "./Components/Post-Thoughts"
+import GetLastPost from "./Components/GetLast-Post"
+import { fecthData, postLikes, postThoughts } from './ServiceAPI/ApiServices'
 
 export const App = () => {
+
+  const [sendMessage, setSendMessage] = useState() 
+
+  const [messages, setMessages] = useState([]) 
+
+  const [error,setError]= useState(false)
+
+  const [lengthError, setLengthError] = useState(false)
+
+  const onChangeText = (e)=>{
+    setSendMessage(e.target.value)
+  }
+ 
+  const submitMessage = async() =>{
+    const {messageSent, error400}  = await postThoughts(sendMessage)
+    setLengthError(error400)
+
+    if(messageSent === true){
+      getThoughts()
+      setSendMessage("")
+      
+    }
+  }
+    const getThoughts = async () => {
+    const apiMessage = await fecthData()
+    setMessages(apiMessage.message)
+    setError(apiMessage.hasError)   
+    
+  }
+  const likesIncrease= async(messageId)=>{
+    const sucess = await postLikes(messageId)
+    if(sucess === true){
+      getThoughts()
+    } 
+  }
+  const autoRefresList = () =>{
+    setInterval(()=>{
+      getThoughts()
+      console.log('refresh-')
+    },5000)
+ }
+ 
+  useEffect(() => {
+    getThoughts()
+    autoRefresList()
+  }, [])
```

Comment:
I'm incredibly impressed with you managing to get the thoughts to auto-update! Stellar! 🤩